profile_bluesky/startup/instrument/devices/blackfly.py

```diff
@@ -76,9 +78,14 @@ def should_save_jpeg(self):
         return _flag_save_sample_image_jpeg_.get() in (1, "Yes")
 
     def take_image(self):
-        yield from bps.stage(self)
-        yield from bps.trigger(self, wait=True)
```

Comment:
Could we have done the same thing by removing the `wait=True` keyword argument?  I don't think so but we could try that.

Reply:
setting `wait=False` does not execute properly but the branch seems to do the right job